docs/source/developer_resources.rst

```diff
@@ -339,6 +339,44 @@ Best Practices
 8. Use the “DO NOT MERGE” label for Pull Requests that should not be merged
 9. When PR has been reviewed and approved, move the ticket/issue to the 'Ready to Deploy to Dev' box in the GitHub project tracker.
 
+Git Naming Conventions
+----------------------
+
+Commit messages should follow the format of
+
+.. code-block:: console
+
+    <type>[optional scope]: <description>
+
+    [optional body]
+
+:code:`type` must be one of the following:
+
+- **docs**: Changes to the documentation (e.g. improving docstring, updating this file, etc)
+- **feat**: Adds a new feature
+- **fix**: A bug fix
+- **refactor**: Changes that don't fix a bug or add a new feature
+- **style**: Changes that don't affect the meaning of code (e.g. whitespace)
+- **test**: Adding or correcting tests
+
+:code:`scope` is optional, and should indicate the general area of the application affected.
```

Comment:
Some try to specify the choices for `scope` as well, but I decided that's probably tough in our case. Models affect views, views affect front end, etc.